src/main/core/support/configuration.rs

```diff
@@ -130,6 +130,37 @@ impl ConfigOptions {
             hosts: config_file.hosts,
         }
     }
+
+    pub fn model_unblocked_syscall_latency(&self) -> bool {
+        self.general.model_unblocked_syscall_latency.unwrap()
+    }
+
+    pub fn max_unapplied_cpu_latency(&self) -> SimulationTime {
+        let nanos = match self.experimental.max_unapplied_cpu_latency {
+            Some(x) => x.convert(units::TimePrefix::Nano).unwrap().value(),
+            // shadow uses a value of 0 as "not set" instead of SIMTIME_INVALID
+            None => 0,
```

Comment:
The C wrapper was removed in a later commit :).

Ultimately I want to implement a default of `SimulationTime::ZERO` here, and return a `SimulationTime` instead of an `Option<SimulationTime>`. I think we originally made the field in the config struct an `Option<SimulationTime>` so that we could tell whether the user provided it or not, but I can't think of why we need that. Should I just change the config struct to be a `SimulationTime` with a default value of `ZERO` instead of an `Option` with a default value of `None`?

Reply:
Correction, the default is already `Some(...)`, which is indeed what we want. I'm still not sure whether I ought to remove the Option wrapper though. I think maybe it had something to do with the processed config we write back out? This way if the user didn't provide an override, the processed config won't have an override either, and a newer version of shadow run with that processed config, with an updated default, will pick up the new value...?

Reply:
Oops, that doesn't make sense either, since we'd only get here if the user explicitly set this flag to None, not if they didn't explicitly set it at all.

So I guess go ahead and change the config field to not be an `Option`?

Reply:
Yeah, the user may provide it in either the cli options or the config options, but probably not both, so there needs to be a possibility for it to be `None`.

> We set it to Some(x) in the defaults though, and I don't think there's any reason to support the user explicitly setting it to an empty string, null, etc. So I just unwrapped the option here. LMK if that seems like the right way to go.

Yeah that's how I've been thinking about it. The documentation states what values/types are allowed for each field, and if they provide a `null` where a `null` isn't allowed, we may as well just unwrap it.